CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ Added:
 - `printGlobalStateSCC` renamed to `printGlobalStateApp`.
 
 - The ` PyASCCache` has been merged to `ASCCache` and is not used anymore.
+- Only use list transaction in executeTx.
```

Comment:
?

Reply:
Oh this was my mistake when I see the file changelog.md on the `dev` branch and put the different to the this commit